internal/config/5_real_config.go

```diff
@@ -27,17 +27,15 @@ func init() {
 	} else {
 		viper.SetDefault("featureFlags.environment", "development")
 
+		// load from main subdirectory
 		viper.AddConfigPath("./configs")
-		viper.SetConfigName("defaults")
+		// load from working directory for integration tests
+		viper.AddConfigPath(".")
```

Comment:
I do not see the link added in this PR, so I do not know for sure, but I think we do not need this if I'm correct.